php/libraries/Utility.class.inc

```diff
@@ -1001,20 +1001,21 @@ class Utility
     /**
      * Get the list of MRI scan type available in the database
      *
+     * @param bool $join_files Whether to perform an inner join on the files table
+     *
      * @return array Array of MRI scan types which exist in the database table
      *               mri_scan_type joined to the files table
      *               array($scan_type_id => $scan_type)
      */
-    static function getScanTypeList(): array
+    static function getScanTypeList($join_files = true): array
     {
         $DB = \Database::singleton();
-
-        $scan_types_DB = $DB->pselect(
-            "SELECT ID, Scan_type 
-             FROM mri_scan_type mri
-                JOIN files f ON (f.AcquisitionProtocolID=mri.ID)",
-            array()
-        );
+        $query = "SELECT ID, Scan_type
+             FROM mri_scan_type mri\n";
```

Comment:
why the `\n` ?

Reply:
It's to preserve space within the query in case the join statement is concatenated. I will replace it with a white space character.